distances.py

```diff
@@ -0,0 +1,193 @@
+import numpy as np
+
+
+def values(hist_dataset, hist_im_query):
+    """
+        Function that verifies the sizes of histograms and matches them
+    Args:
+        hist_dataset: Matrix containing the n dataset histograms data
+        hist_im_query: Matrix containing the test image histogram data
+
+    Returns: Matrix resizing the test image histogram data
+
+    """
+    nim_dataset, nvals_dataset = np.shape(hist_dataset)
+    nims, nvals_query = np.shape(hist_im_query)
+
+    if nvals_query != nvals_dataset:
+        print("Dimensions don't match")
+
+    hist_im_query = np.tile(hist_im_query, (nim_dataset, 1))
```

Comment:
making `hist_im_query = hist_im_query[..., None] ` adds a dimension to array, making possible to broadcast on dataset_hist